kratos/solving_strategies/convergencecriterias/convergence_criteria.h

```diff
@@ -121,154 +108,239 @@ class ConvergenceCriteria
     ///@name Member Variables
     ///@{
 
-
-
     ///@}
     ///@name Operators
     ///@{
 
+    ///@}
+    ///@name Operations
+    ///@{
+
     /**
-     * Get component wise element components
+     * @brief Get component wise element components
+     * @warning Must be defined on the derived classes
+     * @return The RHS element components
      */
     virtual std::vector<TSystemVectorType>&  GetRHS_Element_Components()
```

Comment:
I would propose to remove the GetRHS_Element_Component and GetRHS_Element_Variables (and the same ones for conditions) **as part of a different PR**. To my understanding they are not used anywhere in the code